zh-CN/mapping_dev.zh_CN

```diff
@@ -352,162 +352,162 @@ dlgSgAbout.edt"Version"=版本
+dlgSgBisectStartConfirm.btn"Start Bisect"=开始二分
+dlgSgBisectStartConfirm.fur=在 Git 开始二进制搜索之前，你需要将1个提交标记为好，1个提交为坏。
+dlgSgBisectStartConfirm.hdl=是否应该以错误的提交开始？
+dlgSgBisectStartConfirm.tle=开始二分
+dlgSgBitbucketGenerateToken.edt"Code"=代码
 dlgSgBitbucketGenerateToken.edt"Link"=链接
-dlgSgBitbucketGenerateToken.hdl=Enter the generated code
-dlgSgBitbucketGenerateToken.inf=Authenticate at Bitbucket and enter the generated token
+dlgSgBitbucketGenerateToken.hdl=输入生成的代码
+dlgSgBitbucketGenerateToken.inf=在 Bitbucket 进行身份验证并输入生成的令牌
 dlgSgBitbucketGenerateToken.lbl"Your browser should have opened automatically, let you authenticate with your preferred account at Bitbucket and grant access to SmartGit. If this didn't happen, manually open following link:"=\
- Your browser should have opened automatically, let you authenticate with your preferred account at Bitbucket and grant access to SmartGit. If this didn't happen, manually open following link:
+ 您的浏览器应该已自动打开，让您使用 Bitbucket 上的首选帐户进行身份验证并授予对 SmartGit 的访问权限。如果没有发生这种情况，请手动打开以下链接：
 dlgSgBitbucketGenerateToken.tle=请求访问令牌
 dlgSgBranchAddCheckout.btn"Add Branch && Checkout"=创建分支并检出
 dlgSgBranchAddCheckout.btn"Add Branch"=创建分支
 dlgSgBranchAddCheckout.edt"Branch"=分支
 dlgSgBranchAddCheckout.err"Choose a different branch, the current branch can't be used."=\
- Choose a different branch, the current branch can't be used.
+ 选择不同的分支，当前分支不能使用。
 dlgSgBranchAddCheckout.err"Enter the name of the local branch."=\
- Enter the name of the local branch.
-dlgSgBranchAddCheckout.hdl"Add branch at commit $1"=在$1提交处创建分支
+ 输入本地分支的名称。
+dlgSgBranchAddCheckout.hdl"Add branch at commit $1"=在 $1 提交处创建分支
 dlgSgBranchAddCheckout.hdl"Add branch at current HEAD commit"=\
- 在当前HEAD提交处创建分支
+ 在当前 HEAD 提交处创建分支
 dlgSgBranchAddCheckout.inf=输入要创建的本地分支的名称。
 dlgSgBranchAddCheckout.tle=创建分支
-dlgSgBranchAddCheckoutOverwriteExisting.btn"Overwrite"=Overwrite
-dlgSgBranchAddCheckoutOverwriteExisting.fur=Click 'Cancel' to choose a different branch name.
-dlgSgBranchAddCheckoutOverwriteExisting.hdl=The branch 'temp' already exists. Do you want to overwrite it?
-dlgSgBranchAddCheckoutOverwriteExisting.tle=Add Branch
-dlgSgBranchDeleteLocalConfirmMultiple.btn"Delete"=Delete
-dlgSgBranchDeleteLocalConfirmMultiple.chk"Delete from remote repository"=Delete from remote repository
-dlgSgBranchDeleteLocalConfirmMultiple.chk"Delete tracked branches"=Delete tracked branches
-dlgSgBranchDeleteLocalConfirmMultiple.fur=It might be complicated to restore deleted branches.\n\nYou may only delete a branch from your local list of remote branches, but this may bring back the branch with the next fetch.
-dlgSgBranchDeleteLocalConfirmMultiple.hdl%1=Are you sure to delete $1 local branches?
-dlgSgBranchDeleteLocalConfirmMultiple.tle=Delete
+dlgSgBranchAddCheckoutOverwriteExisting.btn"Overwrite"=覆盖
+dlgSgBranchAddCheckoutOverwriteExisting.fur=单击取消以选择其他分支名称。
+dlgSgBranchAddCheckoutOverwriteExisting.hdl=分支 “temp” 已经存在。你想覆盖它吗？
```

Comment:
I think need to add $1 with temp , @syntevo

Reply:
@Kerwin1202 Thanks, I have fixed this in master now.